src/app/auth/register/personal-info/personal-info.component.ts

```diff
@@ -36,9 +40,13 @@ export class PersonalInfoComponent implements OnInit, OnDestroy {
   constructor(private auth: AuthService, private register: RegisterComponent) {}
 
   ngOnInit() {
+
+  }
+
+  ngAfterViewInit() {
     // listen for input events on the income input and send text to suggestion component after 650 ms of debounce
     this.subs.push(Observable.fromEvent(this.emailInput.nativeElement, 'input')
-      .map(x => x['currentTarget'].value)
+      .map(x =>  x['currentTarget'].value)
```

Comment:
There's two spaces here. Try using the prettier plugin (or something similar) to take care of this for you.

We'll get it automated soon 🤞